core/templates/pages/story-viewer-page/story-viewer-page.directive.html

```diff
@@ -1,255 +1,121 @@
-                  $index !== $ctrl.activeHighlightedIconIndex"
-                        cx="50"
-                        cy="70"
-                        r="50"
-                        id="<['highlight' + $index]>"
-                        fill="#FF8C00"
-                        fill-opacity="0.5"/>
-                <circle ng-show="($index===$ctrl.activeHighlightedIconIndex)?true:false"
-                        cx="50"
-                        cy="70"
-                        r="50"
-                        id="<['highlight' + $index]>"
-                        fill="#A6DACF"
-                        fill-opacity="0.5"/>
-                <circle cx="50"
-                        cy="70"
-                        r="42"
-                        fill="url(#image0<[$index]>)"
-                        stroke="#006553"
-                        stroke-width="2"/>
-              </svg>
-            </a>
-            <img ng-src="<[$ctrl.getStaticImageUrl('/general/mobile_path_segment.svg')]>">
-            <div style="position: fixed; z-index: 100; top: 0; left: 0;">
-              <div ng-class="{'oppia-activity-summary-tile-mobile-background-mask': $ctrl.explorationCardIsShown}"
-                   ng-click="$ctrl.closeOnClickingOutside()">
-                <exploration-summary-tile ng-if="$ctrl.explorationCardIsShown"
-                                          ng-click="$ctrl.onClickStopPropagation($event)"
-                                          exploration-id="$ctrl.isSummaryTileVisible($ctrl.currentNode) ? $ctrl.currentExplorationId : null"
-                                          exploration-title="$ctrl.summaryToPreview.title"
-                                          node-id="$ctrl.currentNodeId"
-                                          last-updated-msec="$ctrl.summaryToPreview.last_updated_msec"
-                                          objective="$ctrl.summaryToPreview.objective"
-                                          category="$ctrl.summaryToPreview.category"
-                                          ratings="$ctrl.summaryToPreview.ratings"
-                                          num-views="$ctrl.summaryToPreview.num_views"
-                                          thumbnail-icon-url="$ctrl.summaryToPreview.thumbnail_icon_url"
-                                          thumbnail-bg-color="$ctrl.summaryToPreview.thumbnail_bg_color"
-                                          is-community-owned="$ctrl.summaryToPreview.community_owned"
-                                          is-collection-preview-tile="true"
-                                          style="position: absolute; z-index: 10;">
-                </exploration-summary-tile>
-              </div>
-            </div>
-          </div>
-        </div>
+                <div class="oppia-story-viewer-card">
+                  <p class="oppia-chapter-title" aria-live="assertive">
+                    Chapter {{$index + 1}}: <[node.getTitle()]>
```

Comment:
Use i18n translation for all learner facing constant texts, though FYI, I've added the translation for 'Chapter' in #9362, so you can use that, or just create a new one and resolve merge conflicts, depending on which PR gets merged first.

Reply:
Done